data/net/minecraft/world/level/levelgen/feature/Feature.mapping

```diff
@@ -2,11 +2,17 @@ CLASS net/minecraft/world/level/levelgen/feature/Feature
 	METHOD <init> (Lcom/mojang/serialization/Codec;)V
 		ARG 1 codec
 	METHOD checkNeighbors (Ljava/util/function/Function;Lnet/minecraft/core/BlockPos;Ljava/util/function/Predicate;)Z
+		COMMENT @return true if any of the six adjacent blocks to the block at {@code pos} return {@code true} to the given filter.
+		ARG 0 adjacentStateAccessor
+			COMMENT An accessor for the adjacent blocks. Essentially {@code level::getBlockState}.
```

Comment:
It is not necessarily an accessor for _adjacent_ blocks, just an accessor for block states given a block position.